src/stream.rs

```diff
@@ -52,17 +54,58 @@ impl ByteExt for u8 {
         matches!(*self, b' ' | b'\t' | b'\n' | b'\r')
     }
 
+    #[inline]
+    fn is_quote(&self) -> bool {
+        matches!(*self, b'\'' | b'"')
+    }
+
     #[inline]
     fn is_letter(&self) -> bool {
         matches!(*self, b'A'..=b'Z' | b'a'..=b'z')
     }
 
     #[inline]
-    fn is_ident(&self) -> bool {
+    fn is_ascii_ident(&self) -> bool {
         matches!(*self, b'0'..=b'9' | b'A'..=b'Z' | b'a'..=b'z' | b'-' | b'_')
     }
 }
 
+trait CssCharExt {
```

Comment:
Let's remove mentions of XML and CSS here. We assume that any XML/CSS related preprocessing was already done before calling `svgtypes`.